corona-warn-app/cwa-app-android
# Booster detail screen (EXPOSUREAPP-11501, EXPOSUREAPP-11592)

## Whats new

Booster card leading to Booster Info Details Screen.

## Jira
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11036)
[Subtask](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11501)
[Subtask](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11592)

## How to test

- Scan any test/certificate
- Go to CCL Configuration and WalletInfo test menu (cheers to @jurajkusnier )
- Select recently scanned person
- Press Add DccWalletInfo button
- Go to person details
- Check the booster card
- Tap on it
- Check the booster info details screen
